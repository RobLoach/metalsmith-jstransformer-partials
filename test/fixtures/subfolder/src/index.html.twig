---
title: Hello, World!
---
<h1>{{ title }}</h1>

<div class="tile-wrapper">
{{ partials['tile']({name:'Hello from Square!'})|raw }}
</div>
