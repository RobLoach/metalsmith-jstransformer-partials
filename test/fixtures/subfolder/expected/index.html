<h1>Hello, World!</h1>

<div class="tile-wrapper">


<div class="card">
  <h2>This is a tile!</h2>
  <h3>Name: Hello from Square!</h3>
  <h3>Last Name: Default Last Name</h3>
</div>

</div>
